frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/index.tsx

```diff
@@ -1,54 +1,67 @@
 import React from 'react';
 import axios, { AxiosResponse } from 'axios';
 import { CategoryStateProps, CategoryUrlParams } from './interfaces';
-import { SubCategory } from '../../../../interfaces';
-import { Col, Row, Table, Typography } from 'antd';
+import { Category, SubCategory } from '../../../../interfaces';
+import { Col, Popconfirm, Row, Table, Typography } from 'antd';
 import styles from './styles.css';
 import { handleApiError } from '../../../../../../services/util/errorHandler';
 import { Link } from 'react-router-dom';
 const { Title } = Typography;
 import { RouteComponentProps } from 'react-router';
 
-const columns = [
-  {
-    title: 'Id',
-    dataIndex: 'id',
-    key: 'id',
-  },
-  {
-    title: 'Name',
-    dataIndex: 'name',
-    key: 'name',
-    // eslint-disable-next-line react/display-name
-    render: (text: string, record: SubCategory) => {
-      const subCategoryName = text
-        .trim()
-        .replace(/\s+|\//g, '-')
-        .toLowerCase();
-      return (
-        <Link
-          to={window.location.pathname + `/${record.id}/${subCategoryName}`}
-        >
-          {text}
-        </Link>
-      );
-    },
-  },
-  {
-    key: 'edit',
-    // eslint-disable-next-line react/display-name
-    render: () => <a>Edit</a>,
-  },
-];
-
 class SubCategories extends React.Component<
   RouteComponentProps<CategoryUrlParams>,
   CategoryStateProps
 > {
   CategoryId: string;
+  columns: any;
   constructor(props: RouteComponentProps<CategoryUrlParams>) {
     super(props);
     this.CategoryId = this.props.match.params.categoryId;
+    this.columns = [
```

Comment:
You can define this outside of the constructor

Reply:
okay